Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -95,14 +92,6 @@ struct PreplannedListItemView: View {
                 self.downloadState = downloadState
             }
         }
-        .task(id: downloadState) {
-            if previousDownloadState == .notDownloaded && downloadState == .downloading {
```

Comment:
It's not ideal to check the status for when a download happens because we should know exactly when it happens in the model. Also this logic should be in a model if possible and not a view. I moved it.